contrib/linux/sensors/file_watch_sensor.py

```diff
@@ -36,13 +33,25 @@ def cleanup(self):
                 pass
 
     def add_trigger(self, trigger):
-        pass
+        if trigger['type'] not in ['linux.file_watch.file_path']:
```

Comment:
Good call - this was probably added after my original trigger with parameters change was implemented so it looks like my comment is obsolete :)

In any case, would be good to double check and verify that is indeed the case.